_posts/patterns/dropdown-menu-ex.html

```diff
@@ -0,0 +1,42 @@
+/**
```

Comment:
How does this get included? I don't see any references to this file. Is there some Jekyll magic going on, or is it not used?